projects/Vula/nss-altfiles.nix

```diff
@@ -0,0 +1,32 @@
+# not exposed because vula uses specific non-release rev and some build flags
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+}: let
+  inherit (lib) licenses maintainers;
+in
+  stdenv.mkDerivation {
+    pname = "nss-altfiles";
```

Comment:
This should be https://codeberg.org/vula/vula_libnss

Reply:
We followed your build setup in [vula_libnss](https://codeberg.org/vula/vula_libnss). We are fetching the git submodule that is used in [vula_libnss](https://codeberg.org/vula/vula_libnss).

Reply:
The upstream doesn't wrap the module in Python as we do and ideally, we want everyone to use the same vula_libnss. It is technically code duplication if a user already had `nss-altfiles` but this is fine given the sensitive nature of the functionality.